src/main/steps/common/page.njk

```diff
@@ -112,4 +112,6 @@
     })(window,document,'script','dataLayer','GTM-WG59GG2');</script>
   <!-- End Google Tag Manager -->
 
+  <script src="https://{{ globals.webchat.avayaClientUrl }}/chat-client/1/app-es2015.js" defer type="module" crossorigin="anonymous"></script>
```

Comment:
what does this script do as we have a webchat script already, curious what the difference is.

Reply:
The avaya-webchat.ts code determines whether to show the link to the webchat or not e.g. if webchat is down, the link will not be shown and some info will be shown instead. app-es2015.js is the code for the actual webchat.